### ISSUE TITLE
media-video/mpv-0.35.1 bump request

### ISSUE THREAD

**Issue by ALTracer** [4re/vapoursynth-portage#120]
## TL;DR
Please consider syncing/bumping stable release version of mpv if this overlay is still active.
## Description
I tried playing a video off YouTube via `mpv https://www.youtube.com/watch?v=` and it got a 854x480 video stream but no audio.
I have `net-misc/yt-dlp-2023.03.04::gentoo` which kinda works.
There was a known issue against yt-dlp and mpv, https://github.com/yt-dlp/yt-dlp/issues/6496, and mpv got the fix merged three weeks ago.
Gentoo version of mpv-0.35.1-r1 carries a patch https://gitweb.gentoo.org/repo/gentoo.git/tree/media-video/mpv/files/mpv-0.35.1-yt-dlp-edl-fragments.patch which is precisely needed to correct this issue.

As I'm still using SVP4 albeit my OpenCL support is broken (no `clinfo` devices with `OCL_ICD_VENDORS=amdocl64.icd`, or `amdgpu-pro-amd64.icd` or `mesa.icd`), I was reluctant to switch from `media-video/mpv-0.35.0::vapoursynth` to `media-video/mpv-0.35.1-r1::gentoo`, I tried building the `9999::vapoursynth` version instead. Luckily, it builds and works with yt-dlp (problem solved, right?) without the patch as it is already contained in master branch.

You may need to add the edl-fragments patch for DASH videos as well in process of syncing the ebuilds with Gentoo master.
Hate to be the one who nags, but I lack the expertise required to provide a pull request for the (bi-monthly?) syncs.

**Comment by 4re** [comment]
No worries, because I use the live ebuild I sometimes miss this kind of updates.
